modules/ROOT/examples/java/app/src/main/java/com/couchbase/code_snippets/Examples.java

```diff
@@ -240,13 +247,14 @@ public void testPreBuiltDatabase() throws IOException {
         // Note: Getting the path to a database is platform-specific.
         // For Android you need to extract it from your
         // assets to a temporary directory and then pass that path to Database.copy()
-        DatabaseConfiguration configuration = new DatabaseConfiguration(context);
+        DatabaseConfiguration configuration = new DatabaseConfiguration();
         if (!Database.exists("travel-sample", context.getFilesDir())) {
             ZipUtils.unzip(getAsset("travel-sample.cblite2.zip"), context.getFilesDir());
             File path = new File(context.getFilesDir(), "travel-sample");
             try {
                 Database.copy(path, "travel-sample", configuration);
-            } catch (CouchbaseLiteException e) {
+            }
+            catch (CouchbaseLiteException e) {
```

Comment:
Is adding a line-break before `catch` a preferred syntax for Android developers? Did you configure Android Studio to show a warning when there isn't a line-break?

I've not seen this syntax before. Anything that adds lines unnecessarily in a code block makes it longer and harder to grasp (personal preference, no science to prove it). So I'm not in favour of this change unless there's a good reason to break the `} catch {` in 2 lines. Being pedantic I know but I believe that stuff really makes a difference considering code snippets get copy pasted often.